unstructured/ingest/interfaces.py

```diff
@@ -160,6 +180,10 @@ def wrapper(self, *args, **kwargs):
 
         return wrapper
 
+    def update_source_metadata(self, **kwargs) -> None:
+        """Sets the SourceMetadata and the  properties for the doc"""
+        pass
```

Comment:
double checking: this is still in progress?

Reply:
ohh woops, this was my bad, I misread this as being a subclass implementation of it. yep, agreed. maybe TODO: make this raise after we implement in existing connectors?